### PR TITLE
Ubiquiti Unifi Identity Desktop version 0.1.13.31

### PR DESCRIPTION
**Pull request by bearmannl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Adding current version of the Unifi Identity Windows application. Unifi officially released an exe installer, but if you check the comments under the release notes they've now also included the msi version for automated install. (silent install triple checked this time!)

Following previous attempt to add the new installer: https://github.com/microsoft/winget-pkgs/pull/132640

Also take note of the alternate DNS domain for the installer download, as described in the previous PR.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135457)

### CODE REVIEW

**Comment by bearmannl** [comment]
> Hello @bearmannl,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

Yes, please check https://github.com/microsoft/winget-pkgs/pull/132640#issuecomment-1880063983 and https://github.com/microsoft/winget-pkgs/pull/57788#issuecomment-1099969928.

**Comment by stephengillie** [comment]
Hi @bearmannl,

Could you add an InstallerUrl?

**Comment by bearmannl** [comment]
> Hi @bearmannl,
> 
> Could you add a PackageUrl?

Done!

**Comment by bearmannl** [comment]
> Hello @bearmannl,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

Yes, please check https://github.com/microsoft/winget-pkgs/pull/132640#issuecomment-1880063983 and https://github.com/microsoft/winget-pkgs/pull/57788#issuecomment-1099969928.

**Comment by bearmannl** [comment]
> > Yes, please check [#132640 (comment)](https://github.com/microsoft/winget-pkgs/pull/132640#issuecomment-1880063983) and [#57788 (comment)](https://github.com/microsoft/winget-pkgs/pull/57788#issuecomment-1099969928).
> 
> Hi @bearmannl,
> 
> > You have to be signed into a UI account to be shown the download links for their new application 'UI Desktop'.
> 
> This package might not be a good fit for addition to the WinGet public manifests. Packages generally have to be available for download from almost any number of users, from almost any number of terrestrial locations, at almost any time. 

Hi @stephengillie,

This is true for the Early Access downloads (I forgot about that), but this release has since been upgraded to general availability which doesn't require a login to view the releasenotes: https://community.ui.com/releases/UniFi-Identity-Windows-0-1-13/0b1449c2-8ba1-40c0-a999-1616395ded88

If I edit the releasenotws URL, does that fix it? Download link remains the same.

File: manifests/u/UbiquitiInc/IdentityDesktop/0.1.13.31/UbiquitiInc.IdentityDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: 0.1.13.31
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.10240.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{0163E59F-332B-4936-86C7-A664306728E9}'
+ReleaseDate: 2024-01-17
+Installers:
+- InstallerUrl: https://fw-download.ubnt.com/data/uid-identity-standard-desktop-app-msi/0d0e-windows-0.1.13-a482dd66-ff91-4835-bf2c-d32fe29f352f.msi
```
Comment (stephengillie) [inline manifests/u/UbiquitiInc/IdentityDesktop/0.1.13.31/UbiquitiInc.IdentityDesktop.installer.yaml:14]:
```suggestion
- InstallerUrl: https://fw-download.ubnt.com/data/uid-identity-standard-desktop-app/1021-windows-0.1.13-ac5d74f6-a672-48ea-b1d6-020f5b5eda35.exe
```

**Comment by stephengillie** [comment]
Thanks for letting me know to check under the `ReleaseNotesUrl`. This might make a better `PackageUrl` than current - I'm unsure. 

I couldn't find the MSI link, only this EXE link. Is the MSI link in another section of the page?

**Comment by bearmannl** [comment]
> Thanks for letting me know to check under the `ReleaseNotesUrl`. This might make a better `PackageUrl` than current - I'm unsure. 
> 
> I couldn't find the MSI link, only this EXE link. Is the MSI link in another section of the page? 

The MSI link is provided by UI Glenn in the comments below (first page). He mentions they will include the MSI package in the actual release files in a future release.

The exe file doesn't have a functional silent install.

**Comment by bearmannl** [comment]
Actually it's the UI Team account. Official source either way.

Comment all the way at the bottom.

File: manifests/u/UbiquitiInc/IdentityDesktop/0.1.13.31/UbiquitiInc.IdentityDesktop.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/u/UbiquitiInc/IdentityDesktop/0.1.13.31/UbiquitiInc.IdentityDesktop.installer.yaml:14]:
```suggestion
- InstallerUrl: https://fw-download.ubnt.com/data/uid-identity-standard-desktop-app-msi/49b8-windows-0.1.13-3377712d-a3ed-498b-bcdc-d56b76783066.msi
```
I got a slightly different path and GUID.

Comment (bearmannl) [reply manifests/u/UbiquitiInc/IdentityDesktop/0.1.13.31/UbiquitiInc.IdentityDesktop.installer.yaml]: Touché. Must've been updated in the meantime.

**Comment by stephengillie** [comment]
This manifest contains a term that is blocked:

> #   - AgreementLabel: EULA

(Automated response - build 660.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**